concourse/tasks/create_admin.yml

```diff
@@ -29,11 +29,22 @@ run:
       uaac target "${UAA_ENDPOINT}"
       uaac token client get admin -s "${UAA_ADMIN_CLIENT_PASS}"
       uaac user add "${NAME}" -p "${PASSWORD}" --emails ignored
+      USERID=$(uaac user get "${NAME}" -a id | awk '{print $2}')
       echo "${NAME}" >admin-creds/username
       echo "${PASSWORD}" >admin-creds/password
+      echo "UserID is ${USERID}"
 
-      uaac member add cloud_controller.admin "${NAME}"
-      uaac member add uaa.admin "${NAME}"
-      uaac member add scim.read "${NAME}"
-      uaac member add scim.write "${NAME}"
-      uaac member add doppler.firehose "${NAME}"
+      set +e
+      add_member(){
+        GID=$(uaac group get $1 -a id | awk '{print $2}')
+        uaac curl -k "/Groups/${GID}/members" -XPOST -H'Content-Type: application/json' \
+                  -d'{"origin":"uaa","type":"USER","value":"'${2}'"}' | tee result | grep -q '201 Created'
+        [ $? != 0 ] && cat result && exit 1
```

Comment:
Could you try simplifying by merging the 2 lines with `grep ... || cat ...` ?
So you could remove `set +e`